e2e/pages/resources/chpl-api/api.po.js

```diff
@@ -0,0 +1,54 @@
+class ApiPage {
+  constructor() {
+    this.elements = {
+      header: 'h1=CHPL API',
+      announcementsController: '#operations-tag-announcements',
+      attestationController: '#operations-tag-attestation',
+      getAttestationEndpoint: '#operations-attestation-getAllPeriods',
+      nameOrganization: '#name-organization',
+      email: '#email',
+      registerButton: '#register-button',
+    };
+  }
+
+  async getTitle() {
+    return (await $(this.elements.header)).getText();
+  }
+
+  async getBodyText() {
+    return (await (await (await $(this.elements.header)).parentElement()).nextElement()).getText();
+  }
+
+  get announcementsController() {
+    return $(this.elements.announcementsController);
+  }
+
+  get attestationController() {
+    return $(this.elements.attestationController);
+  }
+
+  get getAttestationEndpoint() {
+    return $(this.elements.getAttestationEndpoint);
+  }
+
+  get nameOrganization() {
+    return $(this.elements.nameOrganization);
+  }
+
+  get email() {
+    return $(this.elements.email);
+  }
+
+  get registerButton() {
+    return $(this.elements.registerButton);
+  }
+
+  async register() {
+    await (await this.nameOrganization).click();
+    await (await this.nameOrganization).setValue('test');
+    await (await this.email).addValue('test@testorg.com');
+    await (await this.registerButton).click();
+  }
```

Comment:
The values put into the box should be passed in. The PO shouldn't have any idea what the details of the request are, just where to put them

```suggestion
  async register(name, email) {
    await (await this.nameOrganization).click();
    await (await this.nameOrganization).setValue(name);
    await (await this.email).addValue(email);
    await (await this.registerButton).click();
  }
```